elements/chameleon-common/post-install.d/50-ssh-copy-public-keys

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# script for authorizing user's SSH keys
+cat > /etc/copy_user_ssh_keys <<- 'EOM'
+#!/bin/bash
+
+OPENSTACK_VENDOR_DATA='http://169.254.169.254/openstack/latest/vendor_data.json'
+OPENSTACK_VENDOR_DATA_2='http://169.254.169.254/openstack/latest/vendor_data2.json'
+
+JSON_VENDOR_DATA=$(curl -s $OPENSTACK_VENDOR_DATA_2)
+if [ "$JSON_VENDOR_DATA" = '{}' ]; then
+  JSON_VENDOR_DATA=$(curl -s $OPENSTACK_VENDOR_DATA)
+fi
+
+KEYPAIRS=$(python3 -c "import json; print(''.join(obj['keypair']['public_key'] for obj in json.loads('$JSON_VENDOR_DATA', strict=False)['chameleon']['ssh_keypairs']))")
```

Comment:
> I forget if `jq` is available, but it could be used. This is probably simpler though :)

I think `jq` wasn't available when I tried it. 

> This seems to concatenate them all with zero spaces in between? Have you tested w/ multiple keypairs? I would think they'd need to be newline-separated.

The keys end in newlines, and I did test with 2 keys on my account :)